logging/tests/unit/handlers/test__helpers.py

```diff
@@ -63,6 +64,57 @@ def test_valid_context_header(self):
         self.assertEqual(trace_id, expected_trace_id)
 
 
+# webapp2 has not been ported to python3 yet, so do not try to test its
+# functionality there.
+if sys.version_info[0] == 2:
+    class Test_get_trace_id_from_webapp2(unittest.TestCase):
+
+        @staticmethod
+        def create_app():
+            import webapp2
+
+            class GetTraceId(webapp2.RequestHandler):
+                def get(self):
+                    from google.cloud.logging.handlers import _helpers
+
+                    trace_id = _helpers.get_trace_id_from_webapp2()
+                    self.response.content_type = "text/plain"
+                    self.response.out.write(trace_id)
+
+            app = webapp2.WSGIApplication([
+                ('/', GetTraceId),
+            ])
+
+            return app
+
+        def setUp(self):
+            self.app = self.create_app()
```

Comment:
Is there a **need** to do this in `setUp`? If not, (strongly) prefer just creating a test local `app` in each test case.